### ISSUE TITLE
Request for ViT-H backbone configs, weights, and results

### ISSUE THREAD

**Issue by Mori0000** [MiraPurkrabek/ProbPose_code#15]
Hi, thank you for your excellent work on ProbPose!

I noticed that the current repository only provides configs and pre-trained weights for the ViT-Small backbone (ProbPose-small). In the paper, the experiments are also conducted exclusively with ViT-S.

I am interested in evaluating ProbPose with a larger backbone, specifically ViT-Huge (ViT-H), to see how the probabilistic framework scales with model capacity.

Would it be possible to:
1. Share the config files for ProbPose with ViT-H backbone?
2. Provide pre-trained weights if available?
3. If you have already conducted any experiments with ViT-H (or other larger backbones), could you share the results (e.g., mAP / Ex-mAP on COCO/CropCOCO/OCHuman)?
4. If none of the above is available, could you provide guidance on what modifications to the config (e.g., learning rate, batch size, decoder settings) would be needed to train ProbPose with ViT-H?

Thank you for your time!

**Comment by MiraPurkrabek** [comment]
Hi @Mori0000 , 

I assumed you saw the newer [BBoxMaskPose v2](https://github.com/MiraPurkrabek/BBoxMaskPose) work since you commented on it but now I see it is not resolved?

We did not train ProbPose-H directly so I cannot assist there. But there is PMPose-H, which was introduced in [BBoxMaskPose v2](https://github.com/MiraPurkrabek/BBoxMaskPose). It is very similar to ProbPose (also models probability, similar augmentations and outputs) but expects instance mask as an additional input. That makes it much much stronger in crowds, but your detector has to give you a mask...

Here are the pre-trained weights (S/M/L/H):
https://huggingface.co/vrg-prague/BBoxMaskPose/tree/main/PMPose

Let me know if that helped of if you have any questions!
